command_builder.test.ts

```diff
@@ -22,3 +22,19 @@ Deno.test("CommandBuilder.prototype.$", async () => {
     "b",
   );
 });
+
+Deno.test("commandBuilder smoke test", async () => {
+  // TODO: can't reuse the same cmd, probably because pipe hooks stdin of the processes
+  // can this be improved ?
```

Comment:
:+1: Maybe we could just write 
```
const cmd = () => { return $`echo 1`.pipe($`echo "2\n3"`) };
```

Reply:
good workaround :+1: 

I think dax methods in general always allow for  commandBuilder to be copied, because that's advertised in the readme https://github.com/dsherret/dax#commandbuilder (the first example code)